nx/source/services/nifm.c

```diff
@@ -9,6 +9,15 @@
 #include "arm/atomics.h"
 #include "runtime/hosversion.h"
 
+typedef enum {
```

Comment:
Update to match https://switchbrew.org/w/index.php?curid=112&diff=6353&oldid=6349, including order.

Reply:
For the initialization order too ?

Reply:
sure